### PR TITLE
chore: vscode eslint settings

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Opening the project on root directory, config is not automatically found and getting the error below:

image

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://chore-eslint.preview.app.daily.dev

### CODE REVIEW

**Comment by sshanzel** [comment]
@dailydotdev/web-team I've been using vscode again and got this local issue. Let me know if anyone has objections with the file so we can discuss further.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.workingDirectories": [
+    "./packages/shared",
+    "./packages/webapp",
+    "./packages/extension",
+  ]
```
Comment (capJavert) [inline .vscode/settings.json:6]: I don't get this error, would love to jump on a quick call to see whats the issue, because it should work as is.

Comment (capJavert) [reply .vscode/settings.json:6]: @sshanzel are you using the workspace that is provided in `.vscode/apps.code-workspace`, you can open that file manually from "File -> Open Workspace from file" or `code .vscode/apps.code-workspace`

Workspace is configured to include above directories, can you try?

Comment (sshanzel) [reply .vscode/settings.json:6]: Right, let me try and put the config in that direction 👀

Comment (sshanzel) [reply .vscode/settings.json:6]: Worked perfectly fine after that 🥳 🙏

**Pull request closed without merging**